card/mod/05_standard/set/self/signin.rb

```diff
@@ -22,74 +21,83 @@ def default_title_args args
   def default_core_args args={}
     args[:buttons] = button_tag 'Sign in', situation: 'primary'
     if Card.new(type_id: Card::SignupID).ok? :create
-      args[:buttons] += link_to( '...or sign up!', card_path("account/signup"))
+      args[:buttons] += link_to('...or sign up!', card_path('account/signup'))
     end
-    args[:buttons] += raw("<div style='float:right'>#{ view_link 'RESET PASSWORD', :edit, path_opts: {slot: {hide: :toolbar}} }</div>") #FIXME - hardcoded styling
+    args[:buttons] += raw(
+      "<div style='float:right'>" \
+      "#{ view_link 'RESET PASSWORD', :edit,
+                    path_opts: { slot: { hide: :toolbar } } }" \
+      "</div>") # FIXME - hardcoded styling
```

Comment:
Annotation keywords should be all upper case, followed by a colon and a space, then a note describing the problem.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.